src/config/keys.ts

```diff
@@ -6,10 +6,19 @@ import { DEFAULT_THUMBNAIL_SIZE } from './constants';
 
 export const APPS_KEY = 'apps';
 export const ITEMS_KEY = 'items';
+export const SHORT_LINKS_KEY = 'shortLinks';
+export const SHORT_LINK_KEY = 'shortLink';
 export const OWN_ITEMS_KEY = [ITEMS_KEY, 'own'];
 export const ETHERPADS_KEY = 'etherpads';
 export const SUBSCRIPTION_KEY = 'subscriptions';
 
+export const buildShortLinkKey = (alias: string | undefined) => [
+  [SHORT_LINK_KEY, alias],
+];
+export const buildShortLinksItemKey = (id: UUID) => [
+  [SHORT_LINKS_KEY, ITEMS_KEY],
+  id,
+];
```

Comment:
Is there a particular reason you did not use a flat array ? If not please use a flat array
```suggestion
export const buildShortLinksItemKey = (id: UUID) => [
  SHORT_LINKS_KEY, ITEMS_KEY, id
];
```

Reply:
No reason, it is a bad copy/paste I think.